Source/WebKit/UIProcess/Cocoa/ProcessAssertionCocoa.mm

```diff
@@ -318,8 +318,8 @@ - (void)assertion:(RBSAssertion *)assertion didInvalidateWithError:(NSError *)er
 static NSString *runningBoardNameForAssertionType(ProcessAssertionType assertionType)
 {
     switch (assertionType) {
-    case ProcessAssertionType::Suspended:
-        return @"Suspended";
+    case ProcessAssertionType::NearSuspended:
+        return @"Suspended"; // FIXME: This name is confusing since it doesn't cause suspension.
```

Comment:
This *is* confusing! :P 

What does this assertion do? Do you run at background priority?

Reply:
I'll need to ask the RunningBoard team. Miguel told me that this is almost identical to a background assertion so yes to background priority. I think it may drop the jetsam priority lower than the actual background assertion though.

Reply:
Once we find out, it might be worth a follow-on patch to call this "Background" or "BackgroundMac" or "BackgroundNoJetsam" or whatever it really means. (Still, NearSuspended is a definite improvement over Suspended for the time being, since we are definitely not suspended.)